core/common/src/test/java/alluxio/util/CommonUtilsTest.java

```diff
@@ -278,19 +283,26 @@ private void setupShellMocks(String username, List<String> groups) throws IOExce
   @Test
   public void userGroup() throws Throwable {
     String userName = "alluxio-user1";
-    String userGroup1 = "alluxio-user1-group1";
-    String userGroup2 = "alluxio-user1-group2";
-    List<String> userGroups = new ArrayList<>();
-    userGroups.add(userGroup1);
-    userGroups.add(userGroup2);
-    setupShellMocks(userName, userGroups);
+    String userEffectiveGroup1 = "alluxio-user1-effective-group1";
+    String userEffectiveGroup2 = "alluxio-user1-effective-group2";
+    String userAllGroup1 = "alluxio-user1-all-group1";
+    String userAllGroup2 = "alluxio-user1-all-group2";
+    List<String> userEffectiveGroups = new ArrayList<>();
+    List<String> userAllGroups = new ArrayList<>();
+    userEffectiveGroups.add(userEffectiveGroup1);
+    userEffectiveGroups.add(userEffectiveGroup2);
+    userAllGroups.add(userAllGroup1);
```

Comment:
Please add negative case like 
ShellUtils.execCommand(ShellUtils.getEffectiveGroupsForUserCommand("| echo 123 ") to see if that works.

Reply:
Sorry. I'm not quite familiar with `PowerMockito`. These codes are written after my one-hour's quick learning. So, I don't know how to add a negative case.

If possible, I'll do more researches on `PowerMockito`. But I'm much busy now.

And it's welcome that you or someone else to give me a hand to add a negative case.